### DISCUSSION TITLE
Can I make build fail if hydration will complete with mismatches?

### DISCUSSION THREAD

**Discussion by ms264556** [vuejs/vitepress#4191] [Q&A]
I had weird bugs in the interactive part of a page (filtering content using [vue-multiselect](https://vue-multiselect.js.org/)) unless it was wrapped in a `<ClientOnly>` - content was being duplicated instead of replaced.

It turned out to be bad html causing hydration to complete with mismatches.

Is there a (hopefully easy) way to have docs:build fail if the deployed app would need to do hydration fixups?  
Ideally docs:dev would fail too, or at least spit out some warnings, but I'll take what I can get.

**Comment by davestewart** [comment]
It looks like this might be solved now:

- https://github.com/vuejs/vitepress/issues/4189

**Reply by ms264556** [reply]
No, I'm not complaining about bugs in VitePress or Vue.

I want VitePress to report errors in the crappy html I've embedded within markdown pages at build time, rather than quietly repairing them at runtime.
